ddl/ddl_api.go

```diff
@@ -1884,13 +1889,23 @@ func (d *ddl) CreateTableWithInfo(
 		actionType = model.ActionCreateTable
 	}
 
+	createTableQuery, _ := ctx.Value(sessionctx.QueryString).(string)
+	if createTableQuery != "" && ctx.GetSessionVars().StmtCtx.UseShowCreateInDDLJob {
+		var buf bytes.Buffer
+		err = CtorShowCreateTableHook(ctx, tbInfo, nil, &buf)
+		if err != nil {
+			return err
+		}
+		createTableQuery = buf.String()
```

Comment:
CreateTableWithInfo in the same session will continuously walk through this, should set `UseShowCreateInDDLJob` back?

Reply:
Can we always use the show create table?